Doppler.Integrations/Mapper/MapperSubscriber.cs

```diff
@@ -84,6 +85,15 @@ public DopplerSubscriberDto ToDopplerSubscriberDto(IDictionary<string, IList<obj
             };
         }
 
+        public string GetBasicFieldName (string fieldName)
+        {
+            if (!Dictionaries.BasicFieldsNames.TryGetValue(fieldName, out var convertedFieldName))
+            {
+                convertedFieldName = fieldName;
+            }
+            return convertedFieldName;
+        }
```

Comment:
Is this right? 
Wouldn't you want to return the fieldName if it doesn't match any key in the dictionary?

Reply:
It is a negative condition, it was spoken in a call